includes/utilities.php

```diff
@@ -47,8 +47,11 @@ function sanitise_and_normalise_url( $unsafe_url ) {
 		$unsafe_url = add_leading_slash( $unsafe_url );
 	}
 
+	// remove any parameters and trailing slash from the url.
+	$clean_url = rtrim( strtok( $unsafe_url, '?' ), '/' );
```

Comment:
Have we always been stripping the trailing slash? Will this match existing redirects in the database that do have a slash at the end?